src/site/content/en/angular/getting-started-with-server-side-rendering-in-angular/index.md

```diff
@@ -0,0 +1,150 @@
+date: 2019-10-01
+description: |
+  Learn how to set up server-side rendering in your Angular applications.
+authors:
+  - markpieszak
+tags:
+  - angular
+  - performance
+  - ssr
+  - angular-universal
+---
+
+In this post, you'll learn how to use the [Angular command line interface (CLI)](https://cli.angular.io/) to set up server-side rendering (SSR) using Angular Universal. SSR can give you more control over your search engine optimization (SEO) and social media previews and get faster [First Meaningful Paints](/first-meaningful-paint).
+
+{% Aside %}
+
+This post assumes you're already familiar with SSR and its benefits versus traditional client-side rendering (CSR) applications. If you need a refresher, check out Google's [Rendering on the Web](https://developers.google.com/web/updates/2019/02/rendering-on-the-web) article.
+
+{% endAside %}
+
+## What is Angular Universal?
+
+Angular Universal is a nickname for SSR with Angular. SSR is the process of rendering your application on the server rather than in the browser (that is, client-side), which is how most Angular applications have traditionally been rendered.
+
+## Set up SSR for your Angular CLI project
+
+_You can find the code sample from this post [on GitHub](https://github.com/trilonio/angular-universal-tutorial)._
+
+To set up a new Angular application for SSR, run a single command:
+
+```bash
+ng add @nguniversal/express-engine --clientProject angular-universal-tutorial
+```
+
+{% Aside %}
+`clientProject` refers to your application name, which is found in your `angular.json` file. In our [sample code](https://github.com/trilonio/angular-universal-tutorial), the application name is `angular-universal-tutorial`, but make sure to update this to reflect your project's actual name.
+{% endAside %}
+
+Here's what this command does:
+
+1. Creating & modifying project files preparing your application for SSR.
+2. Creates Node.js scripts for dynamic and static SSR.
+3. Installs all necessary dependencies.
+4. Creates new package.json `scripts` that you can use to build SSR.
+
+You now have additional scripts in the sample app that can do dynamic SSR. You'll need to customize these Node.js scripts for production applications, but they're great starting points.
+
+To set up dynamic SSR, run:
+
+```bash
+npm run build:ssr && npm run serve:ssr
+```
+
+For a brand new CLI application, you should be able to run the script above, and everything will work without any problems. But what about your large existing applications?
+
+## Prepare your existing application for SSR
+
+It's important to remember that with SSR your application is being rendered on the server, so any browser-specific functionality will cause errors. Two kinds of code that often cause SSR errors are [global objects](https://developer.mozilla.org/en-US/docs/Glossary/Global_object) and [timeouts](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout) / intervals.
```

Comment:
We really try to avoid slashes in writing. Is it possible to rephrase to avoid the slash between "timeouts" and "intervals"?

Reply:
I'll update it to an `&` instead if that works for you. Apologies, bad writing habits die hard !

Reply:
"and" will work! We don't use ampersands either, haha 😅